### ISSUE TITLE
Remove input pins from "Turn off all digital pins for safety at the end"

### ISSUE THREAD

**Issue by timsainb** [dattalab-6-cam/multicamera_acquisition#42]
Does anyone have a problem with me excluding input pins from this line:  

https://github.com/dattalab-6-cam/multicamera_acquisition/blob/main/microcontroller_firmware/trigger_io/trigger_io.ino#L407C1-L412C14

```
// Turn off all digital pins for safety at the end
for (int i = 0; i < NUM_DIGITAL_PINS; i++)
{
    pinMode(i, OUTPUT);
    digitalWrite(i, LOW);
}
```

The issue being that when you have a second device monitoring the same input, it only works if the cameras are rolling. 

For me a Fed3 is simultaneously being listened to by the teensy, a raspberry pi, and a nidaq board, and this line is messing with that.

**Comment by jonahpearl** [comment]
Makes sense to me. If it's an input pin we shouldn't force it to have one voltage or the other.
